src/lib/leaflet-utils/popup.js

```diff
@@ -26,3 +28,34 @@ export function showLayerInfoPopup ({ map, layerId, unit, selectedLayer, positio
       }
     })
 }
+
+export async function showCombinedLayerInfoPopup ({ coordinates, map }) {
+  const services = await mapConfig.getServices()
+  const HYDRO_ENGINE_URL = services.HYDRO_ENGINE_URL_V2
+  const { lat, lng } = coordinates
+  const url = `${HYDRO_ENGINE_URL}/get_feature_info`
+  const options = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      imageId: 'projects/dgds-gee/glossis/waterlevel/glossis_waterlevel_20200301000000',
+      bbox: {
+        type: 'Point',
+        coordinates: [lat, lng]
+      },
+      band: 'water_level_surge'
+    })
+  }
+
+  fetch(url, options)
+    .then(response => response.json())
+    .then(({ value }) => {
+      if (value !== null) {
+        L.popup()
+          .setLatLng(coordinates)
+          .setContent(`${value}`)
```

Comment:
We could also do `const x = value.toString()` because the value is a Number.

Reply:
yeah let's go for that option (`toString`)